src/sentry/api/endpoints/organization_sdk_updates.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+from datetime import datetime, timedelta
+from sentry.api.bases.organization import OrganizationEndpoint
+
+from itertools import groupby
+from rest_framework.response import Response
+from sentry.sdk_updates import SdkIndexState, SdkSetupState, get_suggested_updates
+
+from sentry.utils.compat import map
+from sentry.snuba import discover
+
+
+def by_sdk_name(sdk):
+    return sdk["sdk.name"]
+
+
+def by_project_id(sdk):
+    return sdk["project.id"]
+
+
+class OrganizationSdkUpdates(OrganizationEndpoint):
+    def get(self, request, organization):
+
+        project_ids = self.get_requested_project_ids(request)
+        projects = self.get_projects(request, organization, project_ids)
+
+        result = discover.query(
```

Comment:
`OrganizationEventsEndpointBase` has a `handle_query_errors` method that will catch and handle most common discover query errors for you

Reply:
Cool, done